applications/secux_hardware_wallet_support_polkadot_transactions_and_staking.md

```diff
@@ -0,0 +1,188 @@
+|      1. | Periodically Meeting                    | Collaborative Efforts and Communication, Polkadot Integration of SecuX SDK support.                                                                                                                                                                                                                                                                                                                                           |
+|      2. | SecuX SDK Polkadot support npm version  | Development of SDK API for Polkadot.js.                                                                                                                                                                                                                                                                                                                                                                                       |
+|      3. | SecuX SE Firmware version number        | Development of Secure Element (SE) Firmware.                                                                                                                                                                                                                                                                                                                                                                                  |
+|      4. | SecuX MCU Firmware version number       | Development of MCU Firmware including transection confirmation, transection details, account balance.                                                                                                                                                                                                                                                                                                                         |
+|      5. | SecuX SDK Polkadot integration tutorial | Provide SDK API for Polkadot.js.                                                                                                                                                                                                                                                                                                                                                                                              |
+|      6. | SecuX Web Wallet Firmware update        | Distribute Polkadot SE and MCU Firmwares.                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
How do you distribute the firmware? Please note that all deliverables should ideally be reusable, so please specify or remove. See also [our FAQ](https://grants.web3.foundation/docs/faq#what-activitiespositions-do-you-fund).

Reply:
Updates will be done through our SecuX web wallet.

Reply:
Can you elaborate what is being delivered here? To reiterate, all deliverables should be reusable and verifiable.

Reply:
revising

Reply:
Are you still working on this, @dannyhuang213?

Reply:
@rus7hex is helping to update it now, thank you for your patience.